Tests/TestingTests/MiscellaneousTests.swift

```diff
@@ -433,15 +433,13 @@ struct MiscellaneousTests {
     #expect(!monomorphicTestFunction.isParameterized)
     let monomorphicTestFunctionTestCases = try #require(monomorphicTestFunction.testCases)
     #expect(monomorphicTestFunctionTestCases.underestimatedCount == 1)
-    #expect(!monomorphicTestFunctionTestCases.isParameterized)
```

Comment:
Why was this expectation removed from the tests?

Reply:
This is calling the `isParameterized` property from the `TestCases` protocol, but since this PR no longer exposes the property of that protocol type this expectation no longer makes sense.

A few lines above, we still have an expectation which calls the same-named `isParameterized` property on `Test`, whose implementation is derived from the former.

Reply:
Fair enough!